.github/workflows/dotnet-core.yml

```diff
@@ -0,0 +1,28 @@
+name: .NET Core
```

Comment:
I am not using Workflow at this point, I do not think this is relevant to have it for such small project.